packages/cmcd-validator-library/src/tests/CMCDValidator/index.js

```diff
@@ -1,3 +1,2 @@
-export { default as testCMCDHeaderValidator } from './testCMCDHeaderValidator.js';
```

Comment:
Is this deletion expected?

Reply:
Yes, it's expected because testCMCDHeaderValidator and testCMCDJsonValidator do not exist anymore.